### DISCUSSION TITLE
How to get all blog list data ?

### DISCUSSION THREAD

**Discussion by Chasen-Zhang** [facebook/docusaurus#10318] [Q&A]
I need some of the blog list data for other common pages, how do I get it, not custom blog list compont

I thought it was:

```

  const data = usePluginData('@docusaurus/plugin-content-blog');
  console.log(data);

```
But not working,  anyone knows?
Help me @slorber

**Comment by slorber** [comment]
Unfortunately it's one of the current limitations of Docusaurus plugin architecture: you can't read one plugin data from another plugin route easily.

This might help work around the problem temporarily: https://github.com/facebook/docusaurus/issues/4138#issuecomment-1703083513

The proper solution would be to introduce React Server Components: https://github.com/facebook/docusaurus/issues/4138#issuecomment-1700580450

**Comment by Chasen-Zhang** [comment]
@slorber Thanks a lot!   
Before running the project or packaging the project, I wrote a node.js script to parse markdown again, and the parsing results is similar:

![image](https://github.com/user-attachments/assets/6a778569-9bbd-4124-94ff-5a9229d2cb3c)
![image](https://github.com/user-attachments/assets/26a4b4dc-7ea6-44f5-abf5-4ac25aa68b65)

It is then introduced as static data, solving the problem for now.
